content/1.7/back-office/orders/orders/shopping-carts-listing.md

```diff
@@ -0,0 +1,77 @@
+- **ID number of Shopping Cart** - specific numeric value, that is registered by Prestashop, every time the cart is created. By default, the ID numbering is set by the ascending arrangement.
+- **Abandoned cart** - if the Cart is not validated through the last checkout step, and the Order is not created, the Cart is statused as Abandoned, and the status becomes _Abandoned Cart_ without any Order ID number.
+- **Customer name and last name** - the unique identifier that shows the Cart creator. If the Cart is created by Guest User, there won't be first and last name in the Shopping Cart, because the Prestashop has not yet transformed the Cart ID into Client ID. If there is registered client, that created the Cart, it will be identified by first and last name.
+- **Total sum of Cart** - numeric value that shows the total amount in the Shopping Cart. It is displayed as a total sum of all the items in the appropriate currency, that is being purchased.
+- **Carrier** - if the user has reached the Carrier step in the Checkout, then Prestashop will register and display the chosen Carrier in the Shopping Cart item.
+- **Date** - the exact time stamp, when the Cart was created.
+- **Online** - the text value, that can be displayed as Yes or No, according to conditioins.
+- **Shop** - if multistores are enabled, Prestashop will show the appropriate shop, where Cart was created.
+- **CTA button View and Delete** - if the Cart is abandoned or the Cart has not been converted to the Orded, the CTA button will be displayed only View. If the Cart has transformed to Order, there will be also a dropdown in View CTA, with the selection of Delete - in order to delete the Cart from Prestashop database.
+
+## The bottom of the Table UI
+
+### Bulk Actions UI
+
+[Bulk Actions in Shopping Cart](static/img/orders-shopping-carts-bulk-actions.png)
+
+A selective UI, where certain global actions can be done, after checking - using checkboxes - in one or multiple Shopping Cart element rows. Bulk Actions has the following functionalities:<br>
+
+- **Select All** - checking this option, Prestashop will select all the checkboxes in the Shopping Carts list.
+- **Unselect All** - checking this option, Prestashop will unselect all the checkboxes in the Shopping Carts list.
+- **Delete selected** - Prestashop will execute a deletion of the selected Shopping Carts rows from the listing. At first, there will be a prompt message informing about deletion.
+
+### Pagination elements UI
+
+The pagination can be selected both in showing the Shopping Carts listed in the table or navigating through the Shopping Cart pages. Shopping Cart lists can be displayed as 20, 50, 100, 300 and 1000 at one viewport screen.
+
+## Other Shopping Carts functionalities
+
+Export carts - CTA button located in the top-right of the page. After clicking, Prestashop generates a specific CSV file for downloading. This CSV file contains all the Shopping Cart data from the table UI.
```

Comment:
```suggestion
Export carts - CTA button located in the top-right of the page. After clicking, it generates and download a CSV file. This CSV file contains all the Shopping Cart data from the table UI.
```

Reply:
Does it download all the date or only the one displayed in the list ?

Reply:
@MatShir it downloads all the shopping carts in the whole list